src/prommis/uky/uky_flowsheet.py

```diff
@@ -2020,16 +2013,17 @@ def function(stream):
             initializer_mix.initialize(m.fs.precip_sx_mixer)
         else:
             print(f"Initializing {stream}")
-            initializer2.initialize(stream)
+            initializer_bt.initialize(stream)
 
     seq.run(m, function)
 
 
-def solve(m):
+def solve(m, solver=None):
     """
     Solve the system with IPOPT.
     """
-    solver = SolverFactory("ipopt")
+    if solver is None:
+        solver = SolverFactory("ipopt")
```

Comment:
Should we use `get_solver` here as the default instead? This comes with some default options that might be useful, and might be important as we eventually switch to the new solver interface in Pyomo.